content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,14 @@
+---
+Title: 'Crazy Eights in  UI/UX'
+Description: 'It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights In UX**
```

Comment:
```suggestion
```
Not necessary to separately add as a title

Reply:
done